pingpong/pingpng.sql

```diff
@@ -0,0 +1,4 @@
+create table quizo (
+    id serial primary key,
+    users text,
+    games integer);
```

Comment:
Consider creating separate tables for users (possibly calling it `players`) and another table called `games`. That way, you can fill the table with multiple users, and each row in a table could have `player1_id`, `player2_id`, `player1_score`, and `player2_score`.

Reply:
so something like this ?

```create table players (
    id serial primary key,
    users text);

create table games (
    id serial primary key,
    player1_id integer,
    player2_id integer,
    player1_score integer,
    player2_score integer);```